new/LIT6024IntroMM.xml

```diff
@@ -0,0 +1,85 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Introductory text to the Miracles of Mary, consisting of a short introduction and the hymn
+                    <ref type="work" corresp="LIT4615Salam"/>, which is also contained within <ref type="work" corresp="LIT1968Mashaf"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2020-06-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following excerpts are taken from <ref type="mss" corresp="BDLaethd20"/></note>
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>በፈቃደ፡ አቡሁ፡ ወበሥምረተ፡ ርእሱ፡ አመ፡ ርእሶ፡ ይሰቅል፨
+                        ወአመ፡ ተዓገሠ፡ ኵሎ፡ ሐፃውንተ፡ ቅዱስ፡ መስቀል፨ ሶበ፡ ሰማዕኪ፡
+                        ማርያም፡ ዘነገሩኪ፡ በቃል። ዘውኅዘ፡ እምአዕይንትኪ፡ ከመ፡ ውኂዘ፡ ማይ፡ ዘይፈለፍል፨
+                        ድንግል፡ ንዒ፡ በምሕረት፡ ወሣህል፨</ab>
+                </div>
+                
+                <div type="textpart" subtype="hymn" xml:id="Hymn" corresp="LIT4615Salam" xml:lang="gez">
+                    <div type="textpart" subtype="incipit">
+                        <ab>
+                            ሃሌ፡ ሉያ፡ በረከታቲሃ፡ ለዛቲ፡ እግዝእትነ፡ ቅድስት፡ ማርያም፡ ወላዲተ፡ አምላክ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ የሃሉ፡ ላዕለ፡ ኵልነ፡ ሕዝብከ፨
+                            ለመስቀልከ፡ እግዚኦ፡ ንፌኑ፡ ስብሐተ፨
+                            ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ 
+                            ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፨
+                        </ab>
+                    </div>
+                    
+                    <div type="textpart" subtype="explicit">
+                        <ab>ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳብ፡ ወሩካቤ፡ በል፨
+                            ወእምዝ፡ ሰላም፡ ለኪ፡ ማርያም፡ ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ፨
+                            ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ፨ ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ፨
```

Comment:
I think we should encode the entire text ሰላም፡ ለኪ፡ (the abbreviation is a scribal habit of the single text witness) or state somehow that this is an abbreviation (`<expan><abbr></abbr><ex></ex>`?)

Reply:
@PietroLiuzzo can `<abbr>` and `<expan>` be used within the text (I suppose so, but this hasn't yet be documentedly done in BM)? In the TEI guidelines examples, both are contained by `<choice>`.

Reply:
@PietroLiuzzo

Reply:
yes @thea-m, @MassimoVilla  is right ([gl page with example](https://betamasaheft.eu/Guidelines/?id=text-encoding)),  no need for `<choice>`